apps/condo/domains/banking/schema/BankTransaction.test.js

```diff
@@ -0,0 +1,557 @@
+
+                const objs = await BankTransaction.getAll(userClient, { id: objCreated.id }, { sortBy: ['updatedAt_DESC'] })
+                expect(objs).toHaveLength(0)
+            })
+
+            test('user can if it is an employee of linked organization', async () => {
+                const [parentOrganization] = await createTestOrganization(admin)
+                const [childOrganization] = await createTestOrganization(admin)
+                const [property] = await createTestProperty(admin, childOrganization)
+                const userClient = await makeClientWithNewRegisteredAndLoggedInUser()
+                await createTestOrganizationLink(admin, parentOrganization, childOrganization)
+                const [role] = await createTestOrganizationEmployeeRole(admin, parentOrganization, {
+                    canManageBankTransactions: true,
+                })
+                await createTestOrganizationEmployee(admin, parentOrganization, userClient.user, role, {})
+                const [integrationContext] = await createTestBankIntegrationContext(admin, bankIntegration, childOrganization)
+                const [account] = await createTestBankAccount(admin, childOrganization, property, {
+                    integrationContext: { connect: { id: integrationContext.id } },
+                })
+                const [contractorAccount] = await createTestBankContractorAccount(admin, childOrganization)
+                const [objCreated] = await createTestBankTransaction(admin, account, contractorAccount, integrationContext, childOrganization)
+
+                const objs = await BankTransaction.getAll(userClient, { id: objCreated.id }, { sortBy: ['updatedAt_DESC'] })
+                expect(objs).toHaveLength(1)
+                expect(objs[0]).toMatchObject(objCreated)
+            })
+
+            test('anonymous can\'t', async () => {
+                const [organization] = await createTestOrganization(admin)
+                const [property] = await createTestProperty(admin, organization)
+                const [integrationContext] = await createTestBankIntegrationContext(admin, bankIntegration, organization)
+                const [account] = await createTestBankAccount(admin, organization, property, {
+                    integrationContext: { connect: { id: integrationContext.id } },
+                })
+                const [contractorAccount] = await createTestBankContractorAccount(admin, organization)
+                const [objCreated] = await createTestBankTransaction(admin, account, contractorAccount, integrationContext, organization)
+
+                await expectToThrowAuthenticationErrorToObjects(async () => {
+                    await BankTransaction.getAll(anonymous, { id: objCreated.id }, { sortBy: ['updatedAt_DESC'] })
+                })
+            })
+        })
+    })
+
+    describe('Validation tests', () => {
+        test('Should have correct dv field (=== 1)', async () => {
```

Comment:
It seems to be redundant. The `DvAndSender` plugin has a validator for this. If you try to create in a test with a different `Dv` value, an error will be thrown. You can see here `packages/keystone/plugins/dvAndSender.js:128`